intercom-java/src/main/java/io/intercom/api/CustomerFirstReply.java

```diff
@@ -0,0 +1,88 @@
+package io.intercom.api;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@SuppressWarnings("UnusedDeclaration")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CustomerFirstReply extends TypedData {
```

Comment:
Could you please reformat this class to use spaces for indentation? It's using a mix of tabs and spaces currently